admin/templates/admin/admin.html

```diff
@@ -21,7 +22,7 @@ <h2 class="card-title">Set maintenance banner</h2>
                             <label for="banner">Banner Details:</label>
                             <textarea class="form-control" name="banner" id="banner" rows="2"
                                 placeholder="Enter system maintenance details..."
-                                required>{{config.get('banner_message')}}</textarea>
+                                required>{{config.get('banner').banner_message}}</textarea>
                         </div>
                         <!-- Checkbox to mark the banner as important -->
```

Comment:
Can we also set the checkbox status based on the current setting?

Reply:
Suggestion adopted.